src/libraries/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -14,5 +14,6 @@ internal static partial class Libraries
         internal const string CompressionNative = "libSystem.IO.Compression.Native";
         internal const string IOPortsNative = "libSystem.IO.Ports.Native";
         internal const string Libdl = "libdl";
+        internal const string HostPolicy = "hostpolicy";
```

Comment:
```suggestion
        internal const string HostPolicy = "libhostpolicy";
```
Or is `hostpolicy` itself correct?

Reply:
It is correct on Windows afaik.

Reply:
Here it's under _Unix_. Still correct? Just want to double-check.

Reply:
It used to be `hostpolicy` for both win and *nix before my PR. But I guess we can accept this change since the actual file is `libhostpolicy.so/dylib` so it should improve discovery a bit).